webapi/src/main/scala/org/knora/webapi/routing/admin/ProjectsRouteADM.scala

```diff
@@ -37,9 +27,21 @@ import org.knora.webapi.routing.RouteUtilADM._
 import org.knora.webapi.routing._
 import org.knora.webapi.slice.admin.api.service.ProjectADMRestService
 
+import pekko.Done
+import pekko.http.scaladsl.model.ContentTypes
+import pekko.http.scaladsl.model.HttpEntity
+import pekko.http.scaladsl.model.headers.ContentDispositionTypes
+import pekko.http.scaladsl.model.headers.`Content-Disposition`
+import pekko.http.scaladsl.server.Directives._
+import pekko.http.scaladsl.server.PathMatcher
+import pekko.http.scaladsl.server.RequestContext
+import pekko.http.scaladsl.server.Route
+import pekko.stream.IOResult
+import pekko.stream.scaladsl.FileIO
+
 final case class ProjectsRouteADM()(
   private implicit val runtime: Runtime[
-    Authenticator with StringFormatter with MessageRelay with ProjectADMRestService
+    org.knora.webapi.routing.Authenticator with StringFormatter with MessageRelay with ProjectADMRestService
```

Comment:
why this change?

Reply:
For some reason the compiler fails if it is not the fully qualified class name.

e.g.:
```
[error] /Users/christian/git/dasch/dsp-api/webapi/src/main/scala/org/knora/webapi/routing/admin/ProjectsRouteADM.scala:296:30: could not find implicit value for parameter r: zio.Runtime[org.knora.webapi.routing.Authenticator with Any with org.knora.webapi.slice.admin.api.service.ProjectADMRestService with Any]
[error]     UnsafeZioRun.runToFuture {
```